examples/main.js

```diff
@@ -0,0 +1,241 @@
+(function(Backbone, $, _) {
+    'use strict';
+
+    // Set driver (optional, but we use localforage here so developers
```

Comment:
Should be "use localStorage" here, I think. Though Firefox devtools has an indexeddb storage inspector that's coming along awesome ;-)